collect/central_collector.go

```diff
@@ -22,7 +22,12 @@ import (
 )
 
 const (
-	cacheEjectBatchSize = 100
+	// TODO: these should be configurable
+	cacheEjectBatchSize          = 100
+	processTracesBatchSize       = 100
+	processTracesBackoffInterval = 200 * time.Microsecond
```

Comment:
These aren't really a backoff since we don't adjust them, it's just a pause, maybe rename them to "pauseDuration" or something?